frontend/src/context/loading/loadingContext.jsx

```diff
@@ -0,0 +1,5 @@
+import { createContext } from "react";
+
+const loadingContext = createContext();
```

Comment:
I'd suggest to change the file extension to tsx as well.
Doing that it will be necessary to explicitly add types.


```suggestion

type ContextProps = {
  loading: boolean,
  handleLoading: () => void,
  handleNotLoading: () => void,
}

const loadingContext = createContext<ContextProps>({ } as ContextProps);
```